src/cpp/session/modules/SessionThemes.R

```diff
@@ -936,13 +980,25 @@
 
    if (!file.remove(themeList[[name]]$fileName))
    {
-      # TODO: give better error when the theme is installed globally.
-      if (file.exists(theme[[name]]$fileName))
+      if (file.exists(themeList[[name]]$fileName))
       {
-         stop(
-            "Unable to remove the speicified theme \"",
-            name,
-            "\". Please check your file system permissions.")
+         justFileName <- basename(themeList[[name]]$fileName)
+         actualPath <- normalizePath(themeList[[name]]$fileName)
```

Comment:
By default `normalizePath` emits warnings if the path doesn't exist -- pass `mustWork = FALSE`. Would also recommend `winslash = "/"` for consistent behavior on Windows.

Reply:
It still works when using `"/"` on windows?

Reply:
Yes, R accepts either.